Bootstraps/AsyncInterface.php

```diff
@@ -1,11 +0,0 @@
-<?php
```

Comment:
Why remove the Async interface? The idea was to give hosted applications access to the same loop for additional async functionality?

Reply:
Because it's entirely unused. With #268 concurrent requests might be unsupported in the future and in that case applications should use their own loop instead of reuse the same loop. Reasons for that are for example that a stopping of the event loop would also stop the outer event loop run.

Reply:
> Because it's entirely unused. 

It's used here: https://github.com/php-pm/php-pm-psr7/blob/d1c225469f1ad2d38c78c571018967cc0186c290/src/Psr7Bridge.php#L34. The idea was to add it to HttpKernel as well.

Reply:
Let's decide on #268 first before continuing the discussion whether it's still something that's needed.